README.md

```diff
@@ -37,22 +37,41 @@ pip install bespokelabs-curator
 
 ```python
 from bespokelabs import curator
-import os
+from datasets import Dataset
+from pydantic import BaseModel, Field
+from typing import List
 
-os.environ['OPENAI_API_KEY'] = 'sk-...' # Set your OpenAI API key here
+# Create a dataset object for the topics you want to create the poems.
+topics = Dataset.from_list([{"topic": "Dreams vs. reality"},
+                            {"topic": "Urban loneliness in a bustling city"},
+                            {"topic": "Beauty of Bespoke Labs's Curator library"}])
 
+# Define a class to encapsulate a list of poems.
+class Poems(BaseModel):
+    poems_list: List[str] = Field(description="A list of poems.")
+
+
+# We define a prompter that generates poems which gets applied to the topics dataset.
 poet = curator.Prompter(
-    prompt_func=lambda: "Write a poem about the beauty of computer science",
+    # The prompt_func takes a row of the dataset as input.
+    # The row is a dictionary with a single key 'topic' in this case.
+    prompt_func=lambda row: f"Write two poems about {row['topic']}.",
     model_name="gpt-4o-mini",
 )
 
-poem = poet()
-print(poem["response"][0])
+poem = poet(topics)
+print(poem.to_pandas())
 ```
+Note that `topics` can be created with `curator.Prompter` as well,
```

Comment:
I think we can also include an example print output from this example in README if ppl just want to see what happens. 

A GIF works well for a demo of curator-viewer in README. Can add later

Reply:
Done